app/components/task-details.hbs

```diff
@@ -1,6 +1,9 @@
 <div class='task-details'>
   <div class='task-details__title-container'>
-    <p class='task-details__title'>{{@task.title}}</p>
+    <a
+      href='{{@taskDetailsURL}}'
```

Comment:
UX improvement: Currently there is no way to know before clicking this link that it will take user to status-site task details, not my-site. A title can be added to the link mentioning it opens task details in status site.

Reply:
It is the same as the status site.

Reply:
Sorry for the confusion, I wanted to mean title attribute of link.